mindmeld/components/dialogue.py

```diff
@@ -935,13 +937,25 @@ async def _end_slot_fill_async(self, request, responder):
         return await self._handler(request, responder)
 
     def _prompt_slot(self, responder, nlr):
+        """Prompts user for missing slot.
+
+        Args:
+            responder (DialogueResponder): responder object.
+            nlr (str): natural language response to prompt for the missing slot.
+        """
+        self._local_form = copy.deepcopy(self._form)
```

Comment:
Is there anyway we can consolidate `_local_form ` and `_form ` into one data structure instead of having two?

Reply:
Removed class dependence, made it into a local variable (response_form).